exercises/clock/clock_test.hs

```diff
@@ -21,29 +21,86 @@ main = exitProperly $ runTestTT $ TestList
 
 clockTests :: [Test]
 clockTests =
-  [ testCase "test constructor and toString" $ do
```

Comment:
I'm assuming these are getting removed because the two components (`toString` and `fromHourMin`) are being tested separately elsewhere, so it would not be necessary to test them together here. I agree.